test/e2e/storage/volume_metrics.go

```diff
@@ -499,10 +500,11 @@ var _ = utils.SIGDescribe(framework.WithSerial(), "Volume metrics", func() {
 	// Test for pv controller metrics, concretely: bound/unbound pv/pvc count.
 	ginkgo.Describe("PVController", func() {
```

Comment:
Those e2e tests only run on some special cloud providers.	

```go
e2eskipper.SkipUnlessProviderIs("gce", "gke", "aws")
```

@jsafrane @msau42 why the e2e tests depend on these provider? Is it possible to use kind cluster and hostpath-csi-driver instead?

And @AndrewSirenko which job can run the new e2e tests? 

All the e2e tests in this file are skipped by `pull-kubernetes-e2e-gce-csi-serial`, `pull-kubernetes-e2e-gce-storage-snapshot` and `pull-kubernetes-e2e-gce-storage-slow`.

Reply:
We can probably now. But for awhile IIRC - mainly it was because of dynamic provisioning and delayed binding etc.